Add initial set of questions (#31)
huserben/scrum-kata: questions/remaining_work.md

---
layout: post
title: "Remaining Work"
parent: Katas
---

> You have items left in the Sprint Backlog at the end of the Sprint. They are not related to the Sprint Goal you select for the next Sprint. What are you doing with those items? Is it wasteful if we start an item and then not follow through with it? How can waste be reduced? Should we just not start items anymore towards the end of the Sprint if we’re not absolutely sure they can be done?

Reviewer: ```suggestion
You have items left in the Sprint Backlog at the end of the Sprint. They are not related to the Sprint Goal you plan to select for the next Sprint. What will you do with those items? Is it wasteful to start an item and then not follow through with it? How can waste be reduced? Should the team not start items towards the end of the Sprint if they are not sure they can be done?
```
